app/src/main/res/values/styles.xml

```diff
@@ -1,27 +1,66 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <style name="app_default_style">
+        <item name="android:textColor">@color/white</item>
+        <item name="android:textSize">@dimen/activity_main_background_text_size</item>
+        <item name="android:textStyle">bold</item>
+        <item name="android:gravity">center</item>
+    </style>
 
-    <style name="button">
-        <item name="android:layout_marginTop">@dimen/fragment_margin_top</item>
-        <item name="android:layout_marginBottom">@dimen/fragment_margin_bottom</item>
+    <style name="main_activity_default_text_view" parent="app_default_style">
```

Comment:
do not create a style for only one property

Reply:
fixed